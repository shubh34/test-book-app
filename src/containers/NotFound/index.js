import PageHeader from '../../components/PageHeader';

const NotFound = () => (
  <div
    style={{
      display: 'flex',
      justifyContent: 'center',
      alignItems: 'center',
      height: '600px'
    }}
  >
    <PageHeader header="Page Not Found" />
  </div>
);

export default NotFound;
