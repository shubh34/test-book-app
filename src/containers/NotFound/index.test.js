import NotFound from '.';
import { setUpComponent } from '../../../test/utils';

test('renders render not page not found', () => {
  const { container } = setUpComponent(NotFound);
  expect(container).toMatchSnapshot();
});
