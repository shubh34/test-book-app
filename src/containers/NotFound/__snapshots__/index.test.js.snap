// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders render not page not found 1`] = `
<div>
  <div
    style="display: flex; justify-content: center; align-items: center; height: 600px;"
  >
    <h3
      class="sc-bdfBwQ irqVJK"
    >
      Page Not Found
    </h3>
  </div>
</div>
`;
